src/Models/Constants/OatConstants.tsx

```diff
@@ -48,15 +48,15 @@ export const getSchemaTypeMenuOptions = (
             }
         };
         switch (x.category) {
+            case 'primitive':
```

Comment:
Any reason why order changed here?

Reply:
well i wanted to swap the order of the items. But I realized i needed to do it a bit further down. Fixed that now.